pilot/pkg/networking/util/util.go

```diff
@@ -758,59 +758,92 @@ func MaybeBuildStatefulSessionFilterConfig(svc *model.Service) *statefulsession.
 	return nil
 }
 
-// MergeTrafficPolicy returns the merged TrafficPolicy for a destination-level and subset-level policy on a given port.
-func MergeTrafficPolicy(original, subsetPolicy *networking.TrafficPolicy, port *model.Port) *networking.TrafficPolicy {
-	if subsetPolicy == nil {
-		return original
+// GetPortLevelTrafficPolicy return the port level traffic policy
```

Comment:
Can you add a comment on what is the bool return value for?

Reply:
sure